terraform/modules/storage/outputs.tf

```diff
@@ -18,6 +18,10 @@ output "phi_storage_bucket" {
   value = google_storage_bucket.phi_storage_bucket.name
 }
 
+output "failed_fhir_conversion_bucket" {
+  value = google_storage_bucket.failed_fhir_conversion_bucket.name
+}
+
```

Comment:
I think data here can be written back to our general phi bucket, but we append the file name with something like `conversion_failures` to specify a "folder" in the bucket's flat namespace. One of the recommendations for the RS DevOps folks was to keep PHI in one location unless there is a need to have it in multiple places. This simplifies thinking through RBAC and other access/permission related issues.

Reply:
Ahhh, makes sense. I'll make that change.